OpenQA/Qemu/DriveDevice.pm

```diff
@@ -98,6 +106,7 @@ sub gen_cmdline ($self) {
         $self->_push_ifdef(\@params, 'bootindex=', $self->bootindex) if (!$path->id || $path->id eq 'path0');
         $self->_push_ifdef(\@params, 'serial=', $self->serial);
         $self->_push_ifdef(\@params, 'num_queues=', $self->num_queues) if (($self->num_queues // -1) != -1);
+        push(@params, 'logical_block_size=' . $self->sector_size . ',physical_block_size=' . $self->sector_size) if $self->sector_size;
```

Comment:
would it make sense to have even a simple sanity check? E.g. `physical_block_size` >= `logical_block_size`? Or does it totally not matter and both can be freely chosen?

Reply:
I think at this level we don't have any restrictions. We simply pass the settings to qemu so let qemu handle any inconsistencies I would say :)